src/bot.js

```diff
@@ -8,8 +8,10 @@ require('dotenv').config();
 const client = new Client({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES, Intents.FLAGS.GUILD_MESSAGE_REACTIONS] });
 // importa as funções de database
 const { checkDatabase } = require('./functions/database.js');
-// importa o loop para checar o horário
-const { loopUtilMidnight } = require('./functions/reset.js');
+// importa a função para trocar a palavra
+const { newWord } = require('./database.js');
+// importa a função para iniciar o timer
+const { runAtMidnight } = require('./functions/runner');
```

Comment:
```suggestion
const { runAtMidnight } = require('./functions/runner.js');
```
Even if it works, it is better to also put the extension.